modules/_node-scripts/configuration/getProjectWebContextPath.mjs

```diff
@@ -0,0 +1,17 @@
+import fs from 'fs/promises';
+
+export default async function getProjectWebContextPath() {
```

Comment:
nit: can we call it `getBndWebContextPath`? It'd differentiate it from when we grab the context path via inferring from package.json